src/coreComponents/physicsSolvers/simplePDE/PhaseFieldDamageFEMKernels.hpp

```diff
@@ -229,28 +229,28 @@ class PhaseFieldDamageKernel :
       {
```

Comment:
we should make this if become an `if constexpr` by adding the dissipation function type as a template parameter for this kernel.